pythonfmu/fmi2slave.py

```diff
@@ -0,0 +1,193 @@
+import datetime
+from abc import ABC, abstractmethod
+from collections import namedtuple
+from io import BytesIO
+from typing import ClassVar, Dict, List, Optional
+from uuid import uuid1
+from xml.etree.ElementTree import Element, SubElement
+
+from .enums import Fmi2Causality
+from .variables import Boolean, Integer, Real, ScalarVariable, String
+
+ModelOptions = namedtuple('ModelOptions', ['name', 'value', 'cli'])
+
+FMI2_MODEL_OPTIONS = [
+    ModelOptions("needsExecutionTool", False, "external-tool"),
```

Comment:
Since Python is not bundled and Python is a tool, `needsExecutionTool` is technically always `True`

Reply:
Sorry about that one. `pyfmi` refuses to open FMU with *needsExecutionTool* = *true*. So I changed it for developing but forget to switch it back.